workflows/sarek.nf

```diff
@@ -335,10 +335,13 @@ workflow SAREK {
     bwa                    = params.bwa                     ? Channel.fromPath(params.bwa).collect()       : PREPARE_GENOME.out.bwa
     bwamem2                = params.bwamem2                 ? Channel.fromPath(params.bwamem2).collect()   : PREPARE_GENOME.out.bwamem2
     dragmap                = params.dragmap                 ? Channel.fromPath(params.dragmap).collect()   : PREPARE_GENOME.out.hashtable
+    sentieon_bwamem        = params.sentieon_bwamem         ? Channel.fromPath(params.sentieon_bwamem).collect() : PREPARE_GENOME.out.sentieon_bwamem
```

Comment:
I think we can use the same `params.bwa`.
but maybe let's add a `params.sentieon` as an extra failsafe, similarly to what we had in sarek pre-3.0.

Reply:
are the indices the same?

Reply:
> are the indices the same?

I got the impression from Don that Sentieon is just a wrapper for the regular bwa index command, but I'm not entirely sure. The md5sums of the produced index files are unchanged, which backs my impression.